openslides/users/static/js/users/site.js

```diff
@@ -850,76 +826,88 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
 
 .controller('GroupListCtrl', [
     '$scope',
+    '$http',
+    'operator',
     'Group',
-    function($scope, Group) {
+    'permissions',
+    function($scope, $http, operator, Group, permissions) {
         Group.bindAll({}, $scope, 'groups');
+        $scope.permissions = permissions;
+
+        $scope.apps = [];
+        var lastApp = '';
+
+        // Create the main clustering with appname->permissions
+        angular.forEach(permissions, function(perm) {
+            var permissionApp = perm.value.split('.')[0]; // get appname
+            var translations = { "agenda": "Agenda",
+                                 "assignments": "Elections",
+                                 "core": "General",
+                                 "mediafiles": "Files",
+                                 "motions": "Motions",
+                                 "users": "Participants" };
+
+            permissionApp = translations[permissionApp];
+
+            if(permissionApp != lastApp) { // new cluster for the app of permission
+                var appCluster = { app_name: permissionApp,
+                                   app_visible: true,
+                                   permissions: [] };
+                lastApp = permissionApp;
+                $scope.apps.push(appCluster);
+            }
+
+            $scope.apps[$scope.apps.length - 1].permissions.push(perm);
+        });
+
+        // has group this permission?
```

Comment:
this comment is hard to understand. Suggestion: "check if the given group contains the given permission"